02-oop-inheritance/keypoints/2_7-12/script.js

```diff
@@ -0,0 +1,209 @@
+/**
+ * Created by MaximilianoDaniel on 17/09/2015.
+ */
+
+var inheritPrototype = function (childClass, parentClass){
+
+    var copyOfParent = Object.create(parentClass.prototype);
+
+    //Is necessary to point to the childClass constructor
+    copyOfParent.constructor = childClass;
+    childClass.prototype = copyOfParent;
+};
+
+function augment( receivingClass, givingClass ) {
+
+    // only provide certain methods
+    // the index starts at two because the first and second arguments are receivingClass and givingClass
+    if ( arguments[2] ) {
+        for ( var i = 2, len = arguments.length; i < len; i++ ) {
```

Comment:
As a good practice, remember to define iterator outside ``for``loop and define variables with more meaningful names.
I would also skip the part where you assign ``arguments.length`` to a variable and just do: ``for (methodStartingIndex; methodStartingIndex < arguments.length; methodStartingIndex += 1)``.